src/Libraries/CoreNodeModelsWpf/CoreNodeModelsWpf.csproj

```diff
@@ -165,60 +86,34 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
-    <Page Include="Controls\ColorPalette.xaml">
-      <SubType>Designer</SubType>
-      <Generator>MSBuild:Compile</Generator>
-    </Page>
-    <Page Include="Controls\DateTimeInputControl.xaml">
-      <SubType>Designer</SubType>
-      <Generator>MSBuild:Compile</Generator>
-    </Page>
-    <Page Include="Controls\DynamoConverterControl.xaml">
+    <Page Include="**\*.xaml">
       <SubType>Designer</SubType>
       <Generator>MSBuild:Compile</Generator>
     </Page>
-    <Page Include="Controls\DoubleSliderSettingsControl.xaml">
-      <Generator>MSBuild:Compile</Generator>
-      <SubType>Designer</SubType>
-    </Page>
-    <Page Include="Controls\DynamoSlider.xaml">
-      <SubType>Designer</SubType>
-      <Generator>MSBuild:Compile</Generator>
-    </Page>
-    <Page Include="Controls\ElementSelectionControl.xaml">
-      <Generator>MSBuild:Compile</Generator>
-      <SubType>Designer</SubType>
-    </Page>
-    <Page Include="Controls\IntegerSliderSettingsControl.xaml">
-      <Generator>MSBuild:Compile</Generator>
-      <SubType>Designer</SubType>
-    </Page>
-  </ItemGroup>
-  <ItemGroup>
-    <Resource Include="Resources\DeprecatedNode.png" />
   </ItemGroup>
   <ItemGroup>
-    <Resource Include="Resources\MissingNode.png" />
```

Comment:
I think this change caused https://jira.autodesk.com/browse/DYN-4697, so I will open a PR to bring this back FYI: @pinzart @mjkkirschner

Reply:
Ooops
I have no idea why I removed that png...

Reply:
@QilongTang DeprecatedNode.png was removed too..

Reply:
@pinzart Yes, but that image is not used since we do not display image for deprecated node I believe. What worries me is that if these two are the only resources missed, currently I cant think of any way to validate that